schema-based-on-diagram.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE medical_histories (
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP NOT NULL,
+  patient_id INTEGER REFERENCES patients(id),
+  status VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
```

Comment:
- [ ] Please make sure all FOREIGN KEYS have indexes created as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/230099273-a918ffaf-8133-47ca-af28-8b90d5b05f2c.png)

Reply:
Thank you @thecodechaser for your feedback. We have implemented all the changes requested.